docusaurus/docs/learn/quickstarts/network/help/upgrade-quickstart-network.md

```diff
@@ -0,0 +1,175 @@
+
+For example, the `ExecStart` section will point to the full path to the old Ziti binary. Update this line to point to
+the path of the newly downloaded Ziti binary.
+
+<Tabs groupId="update-services">
+<TabItem value="28.2" label="v0.28.2 or Later">
+
+The `ExecStart` section will point to the full path to the old Ziti binary. Update this line to point to
+the path of the newly downloaded Ziti binary.
+
+For example, the following portion of the `ExecStart`
+
+```
+.../ziti-v0.28.0/ziti" controller run 
+```
+
+will be updated to the following (assuming a new version of `v0.30.4`).
+
+```
+.../ziti-v0.30.4/ziti" controller run
+```
+
+</TabItem>
+<TabItem value="pre28.2" label="v0.28.1 or Earlier">
+
+In version `v0.28.2`, `ziti-controller`, `ziti-router`, and `ziti-tunnel` were bundled into the `ziti` binary.
+Therefore, in these earlier versions, the `ExecStart` path will not only point to the older version of the Ziti binary
+but the executable used will be `ziti-router` and `ziti-controller`. When upgrading, it is important to change the path
+of the `ExecStart` section to use the new binary (`ziti`).
+
+For example, the following portion of the `ExecStart`
+
+```
+.../ziti-v0.27.0/ziti-controller" run 
+```
+
+will be updated to the following (assuming a new version of `v0.30.4`). Notice the version and executable change.
+
+```
+.../ziti-v0.30.4/ziti" controller run
+```
+
+</TabItem>
+</Tabs>
+
+## Reload and Start Ziti Services
+
+To reflect changes made to the Ziti services, reload the systemd daemon
+
+```
+sudo systemctl daemon-reload
+```
+
+Start Ziti services
+
+```
+sudo systemctl start ziti-controller ziti-router
+```
+
+</TabItem>
+<TabItem value="docker-quickstart" label="Docker Quickstart">
+
+## Docker Users
+
+For Docker users, the upgrade process is much simpler, all the files used for the network are stored in the
+`/persistent` directory of the docker containers. You may back up the files if desired, however the upgrade process
+follows these few instructions;
+
+1. Backup files in `/persistent` if desired
+1. Stop the docker containers but **do not** remove the volume.
+    1. For docker-compose users, use `docker compose down` (do not use the `-v` option)
+    1. For standalone docker users, use `docker stop <container1>, <container2>, ...`
+1. If you used a `.env` file, update the `.env` file to use the desired OpenZiti version.
+1. Pull the desired image
+    1. For docker compose: `docker compose pull`
+    1. For standalone docker: `docker pull openziti/quickstart:<version>`
+1. Start the network back up
+    1. Compose: `docker compose up`
+    1. Standalone: modify the `docker run` command replacing this line `openziti/quickstart:latest` with your desired
+       version
+1. Confirmation of the upgrade can be made
+   by `docker exec <ziti-controller-container> /var/openziti/ziti-bin/ziti version`
+
+</TabItem>
+</Tabs>
+
+## Upgrading Versions <=`v0.28.2`
```

Comment:
since this also appears in the "non-docker" tab, I'd argue that maybe this only belongs on the docker tab? It felt redundant when I looked at it?

![image](https://github.com/openziti/ziti-doc/assets/46322585/2c81dc2c-af3b-4195-8093-f8c24f9feb8d)

Reply:
This (your image snippet) is only in the non-docker tab as it relates to services. The section where this comment is appears in both because it's not part of the main tabs from the beginning of the doc.

This section `Upgrading Versions<=v0.28.2` (which applies to both) states the variable names changed, so if you're upgrading from <= 0.28.2 you will want to update your environment file variable names using the `performMigration` function.